components/apimgt/org.wso2.carbon.apimgt.rest.api.configurations/pom.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0"?>
```

Comment:
Shall we add the License header to this pom as well ?

Reply:
Fixed issue with https://github.com/RenukaFernando/carbon-apimgt/commit/b587c62656dbb084d324e343f09c7f9f8ab1672a